staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset_test.go

```diff
@@ -330,7 +330,9 @@ func (uss *uniformScenarioState) evalTo(lim time.Time, last, expectFair bool, ma
 		var gotFair bool
 		if expectedAverage > 0 {
 			relDiff := (averages[i] - expectedAverage) / expectedAverage
-			gotFair = math.Abs(relDiff) <= margin
```

Comment:
From running the test multiple times, sometimes the value of `relDiff` is something like `-0.15000000066666663`